libfreerdp/core/credssp_auth.c

```diff
@@ -441,6 +441,16 @@ int credssp_auth_authenticate(rdpCredsspAuth* auth)
 		WLog_ERR(TAG, "%s failed with %s [0x%08X]",
 		         auth->server ? "AcceptSecurityContext" : "InitializeSecurityContext",
 		         GetSecurityStatusString(status), status);
+
+		DWORD lastError = GetLastError();
```

Comment:
are you setting this last error code from sspi-rs? we may have to find a better way to pass the error code around that SetLastError/GetLastError

Reply:
1. Yes, I plan to set it in the sspi-rs (in `sspi-rs/ffi`, if be more correct).
2. I was thinking about other ways. But, I didn't find better ways. It'll be good if we have the pointer to the loaded model, but we have only the SSPI functions table.
I had an idea to call `QueryContextAttributesW` with a custom new attribute when a failure occurs in order to receive this code. I'm not sure that is a good idea.